src/stapi_fastapi/models/conformance.py

```diff
@@ -0,0 +1,17 @@
+from pydantic import BaseModel, Field
+
+CORE = "https://stapi.example.com/v0.1.0/core"
+OPPORTUNITIES = "https://stapi.example.com/v0.1.0/opportunities"
+
+
+class Conformance(BaseModel):
+    conforms_to: list[str] = Field(default_factory=list, alias="conformsTo")
+
+    def __init__(
```

Comment:
what's the purpose of this constructor what isn't provided by BaseModel's constructor?

Reply:
ah, I'm pretty new to Pydantic and didn't know to use `serialization_alias` instead of just `alias` so that `conforms_to` would be used in the constructor rather than `conformsTo`.  Fixed.